addons/udes_stock/models/stock_move_line.py

```diff
@@ -455,10 +460,10 @@ def prepare(
                     if move_line.product_uom_qty <= qty_done and remaining_qty:
                         remaining_qty -= move_line.product_uom_qty
                         prod_dict["qty_done"] = move_line.product_uom_qty
-                if measure_quantity:
-                    vals.update({"u_measure_qty": measure_quantity, "u_measure_type": measure_type})
-                prod_dict.update(vals)
-                res[prod_mls] = prod_dict
+                    res[move_line] = prod_dict
+                    if measure_quantity:
+                        vals.update({"u_measure_qty": measure_quantity, "u_measure_type": measure_type})
+                    prod_dict.update(vals)
```

Comment:
I have a similar change on my story in commit https://github.com/unipartdigital/udes-open/pull/948/commits/68ea3137c84024244236f4bdc26ce910c403b4a6

The difference is that in my commit i am copying prod_dict as when used product_dict, depending on the change after first line when was setting res[move_line] = prod_dict for the second move line was changing even the first line. But they are very similar

Reply:
Let's go with your solution @armandcela since it is safer.